pkgs/stdenv/generic/setup.sh

```diff
@@ -1495,17 +1495,7 @@ distPhase() {
 
 showPhaseHeader() {
     local phase="$1"
-    case "$phase" in
-        unpackPhase) echo "unpacking sources";;
-        patchPhase) echo "patching sources";;
-        configurePhase) echo "configuring";;
-        buildPhase) echo "building";;
-        checkPhase) echo "running tests";;
-        installPhase) echo "installing";;
-        fixupPhase) echo "post-installation fixup";;
-        installCheckPhase) echo "running install tests";;
-        *) echo "$phase";;
-    esac
+    echo "started phase $phase"
```

Comment:
```suggestion
    echo "Starting phase $phase"
``

The header is showed before the phase is started, though that should be invinsible to the user.

Reply:
Done. Thanks for feedback. Is it now good to be merged ?